src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-recv-speed-modal/osd-recv-speed-modal.component.html

```diff
@@ -30,7 +30,7 @@
 
         <!-- Customize priority -->
         <div class="form-group row">
-          <div class="offset-sm-6 col-sm-6">
+          <div class="cd-col-form-input cd-col-form-offset">
```

Comment:
You only need `cd-col-form-offset`
```suggestion
          <div class="cd-col-form-offset">
```